index.html

```diff
@@ -11,26 +10,28 @@
       href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
-    <title>Portfolio</title>
+    <link rel="shortcut icon" href="images/logo.png" />
+    <title>Rehema's Portfolio</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   </head>
   <body>
     <div class="page-content">
       <header class="header">
         <nav class="nav">
           <div class="mobile-menu">
             <button type="button">
```

Comment:
- [ ] please make sure that in the 1447px wide screen the hamburger menu does not show up